<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChessPlayer</class>
 <widget class="QMainWindow" name="ChessPlayer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>394</width>
    <height>427</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>黑体</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{background-image: url(:/background1.jpg);}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <layout class="QGridLayout" name="grid"/>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <family>Microsoft YaHei UI</family>
         </font>
        </property>
        <property name="text">
         <string>系统提供</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListView" name="lv">
        <property name="font">
         <font>
          <family>宋体</family>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="font">
           <font>
            <family>Microsoft YaHei UI</family>
           </font>
          </property>
          <property name="text">
           <string>选择</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox">
          <property name="font">
           <font>
            <family>Microsoft YaHei UI</family>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QComboBox{background-image: url(:/newPrefix/button.jpg)}</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <family>Microsoft YaHei UI</family>
           </font>
          </property>
          <property name="text">
           <string>行数：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_row">
          <property name="font">
           <font>
            <family>宋体</family>
           </font>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <family>Microsoft YaHei UI</family>
           </font>
          </property>
          <property name="text">
           <string>列：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_col">
          <property name="font">
           <font>
            <family>宋体</family>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="font">
           <font>
            <family>Microsoft YaHei UI</family>
           </font>
          </property>
          <property name="text">
           <string>棋数：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_num">
          <property name="font">
           <font>
            <family>宋体</family>
           </font>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="font">
           <font>
            <family>Microsoft YaHei UI</family>
           </font>
          </property>
          <property name="text">
           <string>Y： </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_y">
          <property name="font">
           <font>
            <family>宋体</family>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="btn_sure">
          <property name="font">
           <font>
            <family>Microsoft YaHei UI</family>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{background-image: url(:/newPrefix/button.jpg)}</string>
          </property>
          <property name="text">
           <string>确认</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_calc">
          <property name="font">
           <font>
            <family>Microsoft YaHei UI</family>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{background-image: url(:/newPrefix/button.jpg)}</string>
          </property>
          <property name="text">
           <string>计算</string>
          </property>
          <property name="autoRepeatDelay">
           <number>300</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="font">
           <font>
            <family>Microsoft YaHei UI</family>
           </font>
          </property>
          <property name="text">
           <string>out</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_ans">
          <property name="font">
           <font>
            <family>宋体</family>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="btn_exit">
        <property name="font">
         <font>
          <family>Microsoft YaHei UI</family>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{background-image: url(:/newPrefix/button.jpg)}</string>
        </property>
        <property name="text">
         <string>退出</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resource/resource.qrc"/>
 </resources>
 <connections/>
</ui>
